CMakeLists.txt

```diff
@@ -46,6 +46,12 @@ if(NOT CMAKE_HOST_SYSTEM_NAME STREQUAL Windows)
     endif()
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL Linux)
+    # Pass -fno-omit-frame-pointer while compiling for better backtraces
+    add_compile_options("$<$<COMPILE_LANGUAGE:Swift>:SHELL:-Xcc -fno-omit-frame-pointer>")
```

Comment:
Only need one `add_compile_options` call, `COMPILE_LANGUAGE` can take more than one language, and don't need shell escaping.

```cmake
add_compile_options(
  "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-Xcc -fno-omit-frame-pointer>"
  "$<$<COMPILE_LANGUAGE:C,CXX>:-fno-omit-frame-pointer>")
```